Nos/Views/Onboarding/UsernameView.swift

```diff
@@ -0,0 +1,194 @@
+import Dependencies
+import Logger
+import SwiftUI
+
+/// The possible states of ``UsernameView``.
```

Comment:
```suggestion
/// The possible states of `UsernameView`.
```

Reply:
oh did you try this in Xcode? You can Command-click this reference to go to the UsernameView type. Whenever I reference types in my doc comments nowadays, I try to do the double-backticks so they automatically link. Also, Option-click the UsernameViewState type to get its doc comment with this cool underline, which you can then click to view in the documentation window:

<img width="368" alt="Screenshot 2024-10-18 at 3 43 26 PM" src="https://github.com/user-attachments/assets/b82c1eda-9a3a-4be7-8391-f60979091d3d">